assets/install.sh

```diff
@@ -7,47 +7,54 @@
 ##
 
 os="$(uname -s)"
-packages="ruby ruby-dev"
+packages="ruby ruby-dev git"
 
-# install_file TYPE FILENAME
-# TYPE is "rpm", "deb", "solaris", "sh", etc.
-install_file() {
-  echo "Installing the ruby things"
+# install_deps PLATFORM
+# PLATFORM is "redhat", "debian", "freebsd", "arch", etc.
+install_deps() {
+  echo "Installing dependencies"
   case "$1" in
     "redhat")
       echo "installing with yum..."
-      packages="ruby ruby-devel"
+      packages="ruby ruby-devel git"
```

Comment:
Can you change these to be arrays?

```
packages=(ruby ruby-devel git)
```

Reply:
package is a string, right? If you use () that will break lines like

```
apt-get install -y $packages
```

or 

```
pkg install -y $packages
```

Reply:
yes, it needs to be changed to an array, and then the reference needs to be changed to `"${packages[@]}"` so that it can be quoted and you can remove the disable=SC2086

Reply:
@gtmanfred what do you think? If you have other approach to test I can give it a try.

Thanks!

Reply:
Are we not able to run with bash on freebsd?

It would be nice to have the same shell on everything.

Reply:
Well yeah, but:

- Bash is not a default shell
- Bash is not installed by default

So we have to install it in Salt bootstrap or before that, if you are ok I can do the required changes and then do a PR against this to add bash for FreeBSD.

Reply:
let me think about it.